anndata/_core/aligned_mapping.py

```diff
@@ -84,7 +88,10 @@ def parent(self) -> Union["anndata.AnnData", "raw.Raw"]:
     def copy(self):
         d = self._actual_class(self.parent, self._axis)
         for k, v in self.items():
-            d[k] = v.copy()
+            if isinstance(v, AwkArray):
+                d[k] = v
```

Comment:
Do awkward arrays not have an option to copy?

Reply:
It's not a method, though you can `copy.copy(v)` or `copy.deepcopy(v)`.

However, Awkward Arrays are immutable and therefore the distinction between a copy and a view is not user-visible. You can't, for instance, have two views and change an element in one such that another array changes (because you can't change them at all).